_episodes/01-starting-with-data.md

```diff
@@ -208,22 +208,29 @@ We'll talk a bit more about what the different formats mean in a different lesso
 
 ### Useful Ways to View DataFrame objects in Python
 
-There are multiple methods that can be used to summarize and access the data
-stored in DataFrames. Let's try out a few. Note that we call the method by using
-the object name `surveys_df.method`. So `surveys_df.columns` provides an index
-of all of the column names in our DataFrame.
+There are many ways to summarize and access the data stored in DataFrames, 
+using attributes and methods provided by the DataFrame object. Attributes 
```

Comment:
Perhaps:

To access an attribute, use the DataFrame object name followed by the attribute name `df_object.attribute`. Using DataFrame `surveys_df` and attribute `columns`, an index of all the column names in the DataFrame can be accessed with `surveys_df.columns`.